index.html

```diff
@@ -4580,6 +4580,319 @@ <h1>Current Issues</h1>
+        a look-up mechanism is important to establish shared understanding of
+        what a URI identifies. Machines should get RDF data and humans should
+        get a readable representation, such as HTML.
+    </blockquote>
+    <p>
+    Note that, although it is not required that a <a>DID document</a> use an
+    RDF-based representation such as JSON-LD, that format would meet the letter
+    of this W3C recommendation.
+    </p>
+
+    <h2>Can the DID document serve as a representation of the DID subject?</h2>
+    <p>
+    If the <a>DID subject</a> is an information resource that can be retrieved
+    from the Internet, then yes, the <a>DID document</a> can serve as a
+    representation of the <a>DID subject</a>. For example, a data schema that
+    needs a persistent, cryptographically verifiable identifier could be
+    assigned a <a>DID</a>, and its <a>DID document</a> could be used as a
+    standard way to retrieve a representation of that schema.
+    </p>
+
+    <h2>Can existing web resources also be assigned DIDs?</h2>
+    <p>
+    Yes, if the controller of a web page or any other web resource wants to
+    assign it a persistent, cryptographically verifiable identifier, the
+    controller can give it a <a>DID</a>. For example, the author of a blog
+    hosted by a blog hosting company (under that hosting company’s own URL)
+    could create a <a>DID</a> for the blog. In the <a>DID document</a>, the
+    author can include an alsoKnownAs property pointing to the current URL of
+    the blog:
+    </p>
+    <code>
+        “alsoKnownAs”: [“https://myblog.blogging-host.example.com/home”]
+    </code>
+    <p>
+    If the author subsequently moves the blog to a different hosting company
+    (or to the author’s own domain), the author can update the <a>DID document</a>
+    to point to the new URL for the blog:
+    </p>
+    <code>
+        “alsoKnownAs”: [“https://myblog.example.com/”]
+    </code>
+    <p>
+    The <a>DID</a> effectively adds a layer of indirection for the blog URL.
+    This layer of indirection is under the control of the author instead of
+    under external administrative authority such as a blog hosting company. This
+    is how <a>DIDs</a> can effectively function as
+    <a href="https://tools.ietf.org/html/rfc8141">URNs (Uniform Resource
+    Names)</a>—persistent identifiers for information resources whose network
+    location may change over time.
+    </p>
+  </section>
+
+  <section class="appendix">
+    <h1>
+    DID Controllers and DID Subjects
+    </h1>
+    <p>
+    To avoid confusion about the relationship between <a>DID controllers</a> and
+    <a>DID subject</a>s, the DID Working Group has found it helpful to classify
+    <a>DID subject</a>s into two disjoint sets based on their relationship to
+    the <a>DID controller</a>.
+    </p>
+    <h2>Set 1: The DID subject <em>is</em> the DID controller</h2>
+    <p>
+    The first case, shown in figure 4, is the common scenario where the
```

Comment:
link to figure directly, numbering might change

Reply:
@rhiaro I totally agree but I'm not sure how to do that. Can I ask for your help with that after I've processed all these suggestions?